libs/logging-bundle/azure-pipelines.tests.yml

```diff
@@ -1,4 +1,16 @@
 jobs:
   - template: ../../azure-pipelines/jobs/run-tests.yml
     parameters:
-      serviceName: ci-logging-bundle
+      displayName: Tests (Symfony 5.4)
+      serviceName: dev74
+      testCommand: bash -c 'cd libs/logging-bundle && composer install && composer ci'
+      variables:
+        SYMFONY_REQUIRE: "5.4.*"
```

Comment:
Tohe je hlavni point - diky `symfony/flex` jde pouzit `SYMFONY_REQUIRE` k omezeni `composer install` na konkretni verzi Symfony. Znamena to ale, ze `composer install` se musi delat a tady, ne do Docker image (coz je ale ve finale vyhoda, viz commenty vys).

Pokud by se to ujalo a rozsirilo na statni libky, dalo by se cast toho zakomponovat primo do `../../azure-pipelines/jobs/run-tests.yml` a ucesat tak tu konfiguraci.